openaerostruct/geometry/geometry_mesh.py

```diff
@@ -77,12 +77,21 @@ def setup(self):
         # 2. Scale X
 
         val = np.ones(ny)
+        chord_scaling_pos = 0.25  # if no scaling position is specified : chord scaling w.r.t quarter of chord
         if "chord_cp" in surface:
             promotes = ["chord"]
+            if "chord_scaling_pos" in surface:
+                chord_scaling_pos = surface["chord_scaling_pos"]
         else:
+            if "chord_scaling_pos" in surface:
+                print("WARNING: chord_scaling_pos has been specified but no chord design variable available")

```

Comment:
Use the built in warnings package as we do [here](https://github.com/mdolab/openconcept/blob/c92037ed8a03c29030c6ea8c22357fbb279f95ef/openconcept/utilities/math/integrals.py#L602). It's nicer because it allows the user to turn off warnings if desired (among other things)